commands/register.js

```diff
@@ -49,7 +49,10 @@ var channelCheck;
 
 //checks if the user has any charcter, player or NPC, under their control.
 if(userData.possess=="NONE"){
-  aspectChoice = chooseAspect(args[0],client,message);
+  if(args[0]!=undefined)
```

Comment:
ChooseAspect already contains a case for when it's given an undefined argument (the !args[0] check), but its not a bad idea to check twice.

Reply:
Actually, looking more closely, this would create a few problems (namely, players can't register with a random aspect) without solving any. I don't know why the bot crashed when that one player of mine tried to register without providing an aspect, but I don't see how this would've been it.